db/channel_cache_single_test.go

```diff
@@ -32,7 +32,14 @@ func TestDuplicateDocID(t *testing.T) {
 	require.NoError(t, err)
 	defer context.Close(ctx)
 
-	cache := newSingleChannelCache(context, "Test1", 0, (base.NewSyncGatewayStats()).NewDBStats("", false, false, false).Cache())
+	stats, err := base.NewSyncGatewayStats()
+	require.NoError(t, err)
+
+	dbstats, err := stats.NewDBStats("", false, false, false)
+	require.NoError(t, err)
+
+	cache := newSingleChannelCache(context, "Test1", 0, dbstats.Cache())
+	assert.True(t, cache != nil)
```

Comment:
Think there is a not nil check. Not sure if we need to do it but better than `assert.True`